src/toolchain.rs

```diff
@@ -26,37 +27,73 @@ const RUST_CI_COMPONENTS: [(&str, &str); 3] = [
     ("cargo", "cargo-nightly-x86_64-unknown-linux-gnu.tar.xz"),
 ];
 
+lazy_static! {
+    pub static ref MAIN_TOOLCHAIN: Toolchain = Toolchain {
+        source: ToolchainSource::Dist {
```

Comment:
Could we add documentation as to what this tool chain is?

Reply:
Added.